### ISSUE TITLE
NotFoundError: Node.removeChild: The node to be removed is not a child of this node

### ISSUE THREAD

**Issue by aloxe** [decaporg/decap-cms#7278]
**Describe the bug**

I have an existing 11ty site where I added decap-cms with the config described below. It uses an External OAuth client which connects correctly without error message.
The first page of the editor is filled with a long error message containing
```
hl@https://unpkg.com/decap-cms@%5E3.0.0/dist/decap-cms.js:485:2233267 pl@https://unpkg.com/decap-cms@%5E3.0.0/dist/decap-cms.js:485:2232973 hl@https://unpkg.com/decap-cms@%5E3.0.0/dist/decap-cms.js:485:2233922 pl@https://
```
The the content is listed but only the first of the two assets (_articles_, _decouverte_) is shown. I don't see any error on this section of the config file. The error remains when I remove this part.

**Screenshots**

**Applicable Versions:**
 - Decap CMS version: `decap-cms@3.3.3`
 - Git provider: `github`
 - Browser version: `Mozilla/5.0 (X11; Linux x86_64; rv:129.0) Gecko/20100101 Firefox/129.0`

**CMS configuration**
```
backend:
  name: github
  repo: aloxe/reunionweb
  branch: ecrire
  base_url: 
media_folder: src/static/img
collections:
  - name: articles
    label: Articles
    folder: src/pages/articles
    create: true
    fields:
      - label: Layout
        name: layout
        widget: hidden
        default: post
      - label: Title
        name: title
        widget: string
      - label: description
        name: description
        widget: string
      - label: keywords
        name: keywords
        widget: string
      - label: Publish Date
        name: date
        widget: datetime
        time_format: false
        date_format: YYYY-MM
        format: YYYY-MM
      - label: categorie
        name: categorie
        widget: select
        multiple: true
        options:
          - news
          - portails
          - blogs
        default:
          - news
      - label: Body
        name: body
        widget: markdown
    publish: true
    type: folder_based_collection
    sortable_fields:
      - commit_date
      - title
      - date
      - commit_author
      - description
    view_filters: []
    view_groups: []
  - name: decouverte
    label: Découverte
    folder: src/pages/decouverte
    create: true
    fields:
      - label: Layout
        name: layout
        default: decouverte
      - label: Title
        name: title
        widget: string
      - label: description
        name: description
        widget: string
      - label: keywords
        name: keywords
        widget: string
      - label: Body
        name: body
        widget: markdown
    publish: true
    type: folder_based_collection
    sortable_fields:
      - commit_date
      - title
      - commit_author
      - description
    view_filters: []
    view_groups: []
publish_mode: simple
slug:
  encoding: unicode
  clean_accents: false
  sanitize_replacement: "-"
public_folder: /src/static/img
isFetching: false
error: null

```

**Additional context**

**Comment by hip3r** [comment]
@aloxe 
You have some extra code in file https://github.com/aloxe/reunionweb/blob/decap-cms-again/src/assets/public/admin/index.html .
Remove 
```
<script>
      CMS.init();
    </script>
```
and it should work without errors. At least it does on my end.

About collections, `the content is listed but only the first of the two assets (articles, decouverte) is shown`. I copied yml at https://github.com/aloxe/reunionweb/blob/decap-cms-again/src/assets/public/admin/config.yml and it shows all 3 collections
![image](https://github.com/user-attachments/assets/6a2c59e4-7b2b-4e46-8d16-5eab58a5ef4a).


Additionally I would like to ask for your help. We have some issues, where users want to disable git-gateway, like you did. But we are lacking working examples. Issues:  #7297 and https://github.com/decaporg/decap-website/issues/79 .
Would you be able to help with example and/or instructions?

**Comment by aloxe** [comment]
Thanks @hip3r  this works.

Collections are not listing content because content is still nested and in html. I'll work on this later.

Now to your problem, I am not sure I understand you but if you mean by git-gateway that you want to use your own oauth client, this is indeed what I did. I looked at the page https://decapcms.org/docs/external-oauth-clients/ and chose @mcdeck's netlify-cms-oauth-provider-php because it comes with [a blog post](https://www.van-porten.de/blog/2021/01/netlify-auth-provider/) that explains how to just copy it to our own server.

Hope that helps.
